src/applications/mhv-secure-messaging/components/MessageActionButtons/TrashButton.jsx

```diff
@@ -59,8 +59,8 @@ const TrashButton = props => {
         <button
           id="trash-button"
           type="button"
-          className="usa-button-secondary small-screen:vads-u-flex--3 vads-u-display--flex vads-u-flex-direction--row vads-u-justify-content--center vads-u-align-items--center"
-          style={{ minWidth: '100px' }}
+          className="usa-button-secondary small-screen:vads-u-flex--3 vads-u-display--flex vads-u-flex-direction--row vads-u-justify-content--center vads-u-align-items--center vads-u-padding-x--2"
+          style={{ minWidth: '90px' }}
```

Comment:
appears this is a repeating line of code across 3 buttons. Would it make more sense to assign a dedicated class ?